docs/vm/create-windows-vm.md

```diff
@@ -101,3 +101,89 @@ Once all the settings are in place, click on `Create` to create the VM.
+    2. Click `OK` to allow installer to scan this directory for drivers, choose `SUSE Block Driver for Windows` and clock `Next` to load the driver.
+    ![select-virtio-block-driver](assets/select-virtio-block-driver.png)
+    3. Wait for the installer to load up the driver. If you're choosing correct version of driver the `virtio` volumes should be detected once the driver is loaded.
+    ![installer-found-virtio-drive](assets/installer-found-virtio-drive.png)
+    4. Operate this drive at your own discretion.
+
+4. [Optional] If you are using other `virtio` based hardware like network adapter, you will need to manually install those drivers after completed the installation. To install drivers, open VMDP driver disk, and use the installer based on your platform.
+
+The support matrix of VMDP driver pack for Windows are as follows (assumes VMDP CD-ROM drive letter are E):
+
+| Version | Supported | Driver path |
+| :-----: | :-------: | :---------- |
+| Windows 7 | No | `N/A` |
+| Windows Server 2008 | No | `N/A` |
+| Windows Server 2008r2 | No | `N/A` |
+| Windows 8 x86(x64) | Yes | `E:\win8-2012\x86(x64)\pvvx` |
+| Windows Server 2012 x86(x64) | Yes | `E:\win8-2012\x86(x64)\pvvx` |
+| Windows 8.1 x86(x64) | Yes | `E:\win8.1-2012r2\x86(x64)\pvvx` |
+| Windows Server 2012r2 x86(x64) | Yes | `E:\win8.1-2012r2\x86(x64)\pvvx` |
+| Windows 10 x86(x64) | Yes | `E:\win10-server\x86(x64)\pvvx` |
+| Windows Server 2016 x86(x64) | Yes | `E:\win10-server\x86(x64)\pvvx` |
+| Windows Server 2019 x86(x64) | Yes | `E:\win10-server\x86(x64)\pvvx` |
+| Windows 11 x86(x64) | Yes | `E:\win10-2004\x86(x64)\pvvx` |
+| Windows Server 2022 x86(x64) | Yes | `E:\win10-2004\x86(x64)\pvvx` |
+
+!!! note
+    If you need to use other VM template rather than the `windows-iso-image-base-template` we provided, and you still need `virtio` devices, please be sure to provide your own virtio driver disk in order to detect hardware correctly.
+
+## Known Issues
+
+### Windows ISO unable to boot when using EFI mode
+
+When using EFI mode with Windows, you may find out the system booted with other devices like HDD or UEFI shell like the one below:
+
+![efi-shell](assets/efi-shell.png)
+
+That's because Windows will prompt a `Press any key to boot from CD or DVD...` to let user decide whether to boot to installer ISO or not, and it needs human interaction to allow boot from CD or DVD.
+
+![boot-from-cd](assets/boot-from-cd.png)
+
+As a workaround, if the system already booted into UEFI shell, you can type in `reset` to let the system reboot, once the prompt appears you can press any key to let system boot from Windows ISO.
+
+### VM crashes when reserved memory not enough
+
+There is a known issue with Windows VM, when allocated more than 8GiB of memory to Windows VM and not enough reserved memory configured, the VM crashes without warning. This can be temporarily fixed by allocating at least 256MiB of reserved memory to the VM before the installation. 
```

Comment:
```suggestion
There is a known issue with Windows VM, when allocated more than 8GiB of memory to Windows VM and not enough reserved memory configured, the VM crashes without warning. This can be fixed by allocating at least 256MiB of reserved memory to the VM on the advanced tab. 
```